multus-cni e2e test for version `3.8.0` is failing

### Note : multus-cni version `3.7.1`  is working fine
### Possible reason behind the error
Files with `-values.yaml` got [ignored](https://github.com/vmware-tanzu/community-edition/blob/2b55f0da0b712d3a40e028b9f44e540741eeeb33/.gitignore#L62-L63)
<!-- Before creating a new issue please search for any existing related [issues](https://github.com/vmware-tanzu/community-edition/issues?q=is%3Aissue+) before creating a new one. If you find an issue, join the discussion and add emoji or comments with your scenario or subscribe to follow updates. -->

## Bug Report

Multus-cni e2e test is failing for  version `3.8.0` due to missing of file `community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/multus-cni-values.yaml` here https://github.com/vmware-tanzu/community-edition/tree/main/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles.

Logs - 
```
make e2e-test

e2e tests for multus.3.8.0

Running Suite: Multus CNI addon E2E Test Suite
==============================================
Random Seed: 1650875911
Will run 1 of 1 specs

+ tanzu package available list -o json
[
  {
    "display-name": "App-Toolkit package for TCE",
    "latest-version": "0.1.0",
    "name": "app-toolkit.community.tanzu.vmware.com",
    "short-description": "Kubernetes-native toolkit to support application lifecycle"
  },
  {
    "display-name": "Cartographer",
    "latest-version": "0.3.0",
    "name": "cartographer.community.tanzu.vmware.com",
    "short-description": "Kubernetes native Supply Chain Choreographer."
  },
  {
    "display-name": "cert-injection-webhook",
    "latest-version": "0.1.1",
    "name": "cert-injection-webhook.community.tanzu.vmware.com",
    "short-description": "The Cert Injection Webhook injects CA certificates and proxy environment variables into pods"
  },
  {
    "display-name": "cert-manager",
    "latest-version": "1.8.0",
    "name": "cert-manager.community.tanzu.vmware.com",
    "short-description": "Certificate management"
  },
  {
    "display-name": "contour",
    "latest-version": "1.20.1",
    "name": "contour.community.tanzu.vmware.com",
    "short-description": "An ingress controller"
  },
  {
    "display-name": "external-dns",
    "latest-version": "0.10.0",
    "name": "external-dns.community.tanzu.vmware.com",
    "short-description": "This package provides DNS synchronization functionality."
  },
  {
    "display-name": "fluent-bit",
    "latest-version": "1.7.5",
    "name": "fluent-bit.community.tanzu.vmware.com",
    "short-description": "Fluent Bit is a fast Log Processor and Forwarder"
  },
  {
    "display-name": "Flux Source Controller",
    "latest-version": "0.21.2",
    "name": "fluxcd-source-controller.community.tanzu.vmware.com",
    "short-description": "The source-controller is a Kubernetes operator, specialised in artifacts acquisition from external sources such as Git, Helm repositories and S3 buckets."
  },
  {
    "display-name": "gatekeeper",
    "latest-version": "3.7.1",
    "name": "gatekeeper.community.tanzu.vmware.com",
    "short-description": "policy management"
  },
  {
    "display-name": "grafana",
    "latest-version": "7.5.11",
    "name": "grafana.community.tanzu.vmware.com",
    "short-description": "Visualization and analytics software"
  },
  {
    "display-name": "harbor",
    "latest-version": "2.4.2",
    "name": "harbor.community.tanzu.vmware.com",
    "short-description": "OCI Registry"
  },
  {
    "display-name": "knative-serving",
    "latest-version": "1.0.0",
    "name": "knative-serving.community.tanzu.vmware.com",
    "short-description": "Knative Serving builds on Kubernetes to support deploying and serving of applications and functions as serverless containers"
  },
  {
    "display-name": "kpack dependencies",
    "latest-version": "0.0.9",
    "name": "kpack-dependencies.community.tanzu.vmware.com",
    "short-description": "Dependencies in the form of Buildpacks and Stacks for the kpack package"
  },
  {
    "display-name": "kpack",
    "latest-version": "0.5.2",
    "name": "kpack.community.tanzu.vmware.com",
    "short-description": "kpack builds application source code into OCI compliant images using Cloud Native Buildpacks"
  },
  {
    "display-name": "Flux Kustomize Controller",
    "latest-version": "0.21.1",
    "name": "kustomize-controller.fluxcd.community.tanzu.vmware.com",
    "short-description": "Kustomize controller is one of the components in GitOps toolkit."
  },
  {
    "display-name": "local-path-storage",
    "latest-version": "0.0.22",
    "name": "local-path-storage.community.tanzu.vmware.com",
    "short-description": "This package provides local path node storage and primarily supports RWO AccessMode."
  },
  {
    "display-name": "multus-cni",
    "latest-version": "3.8.0",
    "name": "multus-cni.community.tanzu.vmware.com",
    "short-description": "This package provides the ability for enabling attaching multiple network interfaces to pods in Kubernetes"
  },
  {
    "display-name": "prometheus",
    "latest-version": "2.27.0-1",
    "name": "prometheus.community.tanzu.vmware.com",
    "short-description": "A time series database for your metrics"
  },
  {
    "display-name": "velero",
    "latest-version": "1.8.0",
    "name": "velero.community.tanzu.vmware.com",
    "short-description": "Disaster recovery capabilities"
  },
  {
    "display-name": "whereabouts",
    "latest-version": "0.5.1",
    "name": "whereabouts.community.tanzu.vmware.com",
    "short-description": "A CNI IPAM plugin that assigns IP addresses cluster-wide"
  }
]
+ tanzu package available list multus-cni.community.tanzu.vmware.com -o json
[
  {
    "name": "multus-cni.community.tanzu.vmware.com",
    "released-at": "2021-06-04 23:30:00 +0530 IST",
    "version": "3.7.1"
  },
  {
    "name": "multus-cni.community.tanzu.vmware.com",
    "released-at": "0001-01-01 00:00:00 +0000 UTC",
    "version": "3.8.0"
  }
]
Multus CNI Addon E2E Test 
  Check macvlan interfaces status
  /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go:98
STEP: Install Multus CNI Package and tests' resources
+ tanzu package install multus-cni-pkg --package-name multus-cni.community.tanzu.vmware.com --version 3.7.1 -n default -f /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/multus-cni-values.yaml
Error: failed to read from data values file '/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/multus-cni-values.yaml': open /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/multus-cni-values.yaml: no such file or directory
Error: exit status 1


After test fails, theses resources are related.
+ kubectl get app multus-cni-pkg -n default -o jsonpath={.status}
Error from server (NotFound): apps.kappctrl.k14s.io "multus-cni-pkg" not found

STEP: Should delete Multus CNI test temporary resources
+ kubectl delete -f /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/simple-macvlan.yaml
Error from server (NotFound): error when deleting "/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/simple-macvlan.yaml": the server could not find the requested resource (delete network-attachment-definitions.k8s.cni.cncf.io macvlan-config)
Error from server (NotFound): error when deleting "/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/simple-macvlan.yaml": pods "macvlan-worker" not found


• Failure in Spec Setup (BeforeEach) [0.712 seconds]
Multus CNI Addon E2E Test
/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go:26
  Check macvlan interfaces status [BeforeEach]
  /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go:98

  Unexpected error:
      <*errors.errorString | 0xc000111a70>: {
          s: "Error: failed to read from data values file '/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/multus-cni-values.yaml': open /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/multus-cni-values.yaml: no such file or directory\nError: exit status 1\n\nexit status: 1",
      }
      Error: failed to read from data values file '/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/multus-cni-values.yaml': open /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/multus-cni-values.yaml: no such file or directory
      Error: exit status 1
      
      exit status: 1
  occurred

  /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go:37
------------------------------
Multus CNI Addon E2E Test 
  Check macvlan interfaces status
  /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go:98
STEP: Install Multus CNI Package and tests' resources
+ tanzu package install multus-cni-pkg --package-name multus-cni.community.tanzu.vmware.com --version 3.7.1 -n default -f /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/multus-cni-values.yaml
Error: failed to read from data values file '/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/multus-cni-values.yaml': open /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/multus-cni-values.yaml: no such file or directory
Error: exit status 1


After test fails, theses resources are related.
+ kubectl get app multus-cni-pkg -n default -o jsonpath={.status}
Error from server (NotFound): apps.kappctrl.k14s.io "multus-cni-pkg" not found

STEP: Should delete Multus CNI test temporary resources
+ kubectl delete -f /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/simple-macvlan.yaml
Error from server (NotFound): error when deleting "/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/simple-macvlan.yaml": the server could not find the requested resource (delete network-attachment-definitions.k8s.cni.cncf.io macvlan-config)
Error from server (NotFound): error when deleting "/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/simple-macvlan.yaml": pods "macvlan-worker" not found


• Failure in Spec Setup (BeforeEach) [0.590 seconds]
Multus CNI Addon E2E Test
/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go:26
  Check macvlan interfaces status [BeforeEach]
  /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go:98

  Unexpected error:
      <*errors.errorString | 0xc000111fa0>: {
          s: "Error: failed to read from data values file '/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/multus-cni-values.yaml': open /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/multus-cni-values.yaml: no such file or directory\nError: exit status 1\n\nexit status: 1",
      }
      Error: failed to read from data values file '/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/multus-cni-values.yaml': open /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/multus-cni-values.yaml: no such file or directory
      Error: exit status 1
      
      exit status: 1
  occurred

  /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go:37
------------------------------


Summarizing 2 Failures:

[Fail] Multus CNI Addon E2E Test [BeforeEach] Check macvlan interfaces status 
/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go:37

[Fail] Multus CNI Addon E2E Test [BeforeEach] Check macvlan interfaces status 
/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go:37

Ran 1 of 1 Specs in 4.200 seconds
FAIL! -- 0 Passed | 1 Failed | 0 Flaked | 0 Pending | 0 Skipped
--- FAIL: TestMultusCNIE2E (4.20s)
FAIL

Ginkgo ran 1 suite in 6.125325378s
Test Suite Failed
make: *** [e2e-test] Error 1
```


<!-- Please concisely describe the problem you are having. -->

## Expected Behavior

<!-- Describe what you expected to happen. -->
Test should pass.
## Steps to Reproduce the Bug
```
cd addons/packages/multus-cni/3.8.0/test && make e2e-test
```
<!-- Describe steps to reproduce here and how reproducible this issue is (occasionally, often, always) -->

## Screenshots or additional information and context

<!-- If screenshots or snippets can be provided, please do so here. Otherwise please add additional context if necessary. -->

## Environment Details

* Build version (`tanzu version`): `v0.12.0-rc.2`
* Deployment (Managed/Unmanaged cluster): Unmanaged cluster
* Infrastructure Provider (Docker/AWS/Azure/vSphere): Docker
* Operating System (client): Mac

## Diagnostics and log bundle
<!-- Collecting diagnostics and log bundles can help troubleshoot your issue -->

After adding the missing file the test is failing with below logs
```
make e2e-test                           

e2e tests for multus.3.8.0

Running Suite: Multus CNI addon E2E Test Suite
==============================================
Random Seed: 1650882655
Will run 1 of 1 specs

+ tanzu package available list -o json
[
  {
    "display-name": "App-Toolkit package for TCE",
    "latest-version": "0.1.0",
    "name": "app-toolkit.community.tanzu.vmware.com",
    "short-description": "Kubernetes-native toolkit to support application lifecycle"
  },
  {
    "display-name": "Cartographer",
    "latest-version": "0.3.0",
    "name": "cartographer.community.tanzu.vmware.com",
    "short-description": "Kubernetes native Supply Chain Choreographer."
  },
  {
    "display-name": "cert-injection-webhook",
    "latest-version": "0.1.1",
    "name": "cert-injection-webhook.community.tanzu.vmware.com",
    "short-description": "The Cert Injection Webhook injects CA certificates and proxy environment variables into pods"
  },
  {
    "display-name": "cert-manager",
    "latest-version": "1.8.0",
    "name": "cert-manager.community.tanzu.vmware.com",
    "short-description": "Certificate management"
  },
  {
    "display-name": "contour",
    "latest-version": "1.20.1",
    "name": "contour.community.tanzu.vmware.com",
    "short-description": "An ingress controller"
  },
  {
    "display-name": "external-dns",
    "latest-version": "0.10.0",
    "name": "external-dns.community.tanzu.vmware.com",
    "short-description": "This package provides DNS synchronization functionality."
  },
  {
    "display-name": "fluent-bit",
    "latest-version": "1.7.5",
    "name": "fluent-bit.community.tanzu.vmware.com",
    "short-description": "Fluent Bit is a fast Log Processor and Forwarder"
  },
  {
    "display-name": "Flux Source Controller",
    "latest-version": "0.21.2",
    "name": "fluxcd-source-controller.community.tanzu.vmware.com",
    "short-description": "The source-controller is a Kubernetes operator, specialised in artifacts acquisition from external sources such as Git, Helm repositories and S3 buckets."
  },
  {
    "display-name": "gatekeeper",
    "latest-version": "3.7.1",
    "name": "gatekeeper.community.tanzu.vmware.com",
    "short-description": "policy management"
  },
  {
    "display-name": "grafana",
    "latest-version": "7.5.11",
    "name": "grafana.community.tanzu.vmware.com",
    "short-description": "Visualization and analytics software"
  },
  {
    "display-name": "harbor",
    "latest-version": "2.4.2",
    "name": "harbor.community.tanzu.vmware.com",
    "short-description": "OCI Registry"
  },
  {
    "display-name": "knative-serving",
    "latest-version": "1.0.0",
    "name": "knative-serving.community.tanzu.vmware.com",
    "short-description": "Knative Serving builds on Kubernetes to support deploying and serving of applications and functions as serverless containers"
  },
  {
    "display-name": "kpack dependencies",
    "latest-version": "0.0.9",
    "name": "kpack-dependencies.community.tanzu.vmware.com",
    "short-description": "Dependencies in the form of Buildpacks and Stacks for the kpack package"
  },
  {
    "display-name": "kpack",
    "latest-version": "0.5.2",
    "name": "kpack.community.tanzu.vmware.com",
    "short-description": "kpack builds application source code into OCI compliant images using Cloud Native Buildpacks"
  },
  {
    "display-name": "Flux Kustomize Controller",
    "latest-version": "0.21.1",
    "name": "kustomize-controller.fluxcd.community.tanzu.vmware.com",
    "short-description": "Kustomize controller is one of the components in GitOps toolkit."
  },
  {
    "display-name": "local-path-storage",
    "latest-version": "0.0.22",
    "name": "local-path-storage.community.tanzu.vmware.com",
    "short-description": "This package provides local path node storage and primarily supports RWO AccessMode."
  },
  {
    "display-name": "multus-cni",
    "latest-version": "3.8.0",
    "name": "multus-cni.community.tanzu.vmware.com",
    "short-description": "This package provides the ability for enabling attaching multiple network interfaces to pods in Kubernetes"
  },
  {
    "display-name": "prometheus",
    "latest-version": "2.27.0-1",
    "name": "prometheus.community.tanzu.vmware.com",
    "short-description": "A time series database for your metrics"
  },
  {
    "display-name": "velero",
    "latest-version": "1.8.0",
    "name": "velero.community.tanzu.vmware.com",
    "short-description": "Disaster recovery capabilities"
  },
  {
    "display-name": "whereabouts",
    "latest-version": "0.5.1",
    "name": "whereabouts.community.tanzu.vmware.com",
    "short-description": "A CNI IPAM plugin that assigns IP addresses cluster-wide"
  }
]
+ tanzu package available list multus-cni.community.tanzu.vmware.com -o json
[
  {
    "name": "multus-cni.community.tanzu.vmware.com",
    "released-at": "2021-06-04 23:30:00 +0530 IST",
    "version": "3.7.1"
  },
  {
    "name": "multus-cni.community.tanzu.vmware.com",
    "released-at": "0001-01-01 00:00:00 +0000 UTC",
    "version": "3.8.0"
  }
]
Multus CNI Addon E2E Test 
  Check macvlan interfaces status
  /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go:98
STEP: Install Multus CNI Package and tests' resources
+ tanzu package install multus-cni-pkg --package-name multus-cni.community.tanzu.vmware.com --version 3.7.1 -n default -f /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/multus-cni-values.yaml

+ kubectl apply -f /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/cni-install.yaml
configmap/cni-install-sh created
daemonset.apps/install-cni-plugins created

+ kubectl -n default get packageinstalls multus-cni-pkg -o jsonpath={.status.conditions[?(@.type == 'ReconcileSucceeded')].status}
True
+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found


After test fails, theses resources are related.
+ kubectl get app multus-cni-pkg -n default -o jsonpath={.status}
{"conditions":[{"status":"True","type":"ReconcileSucceeded"}],"consecutiveReconcileSuccesses":1,"deploy":{"exitCode":0,"finished":true,"startedAt":"2022-04-25T10:31:12Z","stdout":"Target cluster 'https://10.96.0.1:443' (nodes: ucclust-control-plane)\nChanges\nNamespace    Name                                            Kind                      Conds.  Age  Op      Op st.  Wait to    Rs  Ri\n(cluster)    multus                                          ClusterRole               -       -    create  -       reconcile  -   -\n^            multus                                          ClusterRoleBinding        -       -    create  -       reconcile  -   -\n^            network-attachment-definitions.k8s.cni.cncf.io  CustomResourceDefinition  -       -    create  -       reconcile  -   -\nkube-system  kube-multus-ds-amd64                            DaemonSet                 -       -    create  -       reconcile  -   -\n^            multus                                          ServiceAccount            -       -    create  -       reconcile  -   -\n^            multus-cni-config                               ConfigMap                 -       -    create  -       reconcile  -   -\nOp:      6 create, 0 delete, 0 update, 0 noop\nWait to: 6 reconcile, 0 delete, 0 noop\n10:31:12AM: ---- applying 5 changes [0/6 done] ----\n10:31:12AM: create configmap/multus-cni-config (v1) namespace: kube-system\n10:31:12AM: create serviceaccount/multus (v1) namespace: kube-system\n10:31:12AM: create clusterrole/multus (rbac.authorization.k8s.io/v1) cluster\n10:31:12AM: create clusterrolebinding/multus (rbac.authorization.k8s.io/v1) cluster\n10:31:13AM: create customresourcedefinition/network-attachment-definitions.k8s.cni.cncf.io (apiextensions.k8s.io/v1) cluster\n10:31:13AM: ---- waiting on 5 changes [0/6 done] ----\n10:31:13AM: ok: reconcile clusterrolebinding/multus (rbac.authorization.k8s.io/v1) cluster\n10:31:13AM: ok: reconcile serviceaccount/multus (v1) namespace: kube-system\n10:31:13AM: ok: reconcile clusterrole/multus (rbac.authorization.k8s.io/v1) cluster\n10:31:13AM: ok: reconcile customresourcedefinition/network-attachment-definitions.k8s.cni.cncf.io (apiextensions.k8s.io/v1) cluster\n10:31:13AM: ok: reconcile configmap/multus-cni-config (v1) namespace: kube-system\n10:31:13AM: ---- applying 1 changes [5/6 done] ----\n10:31:13AM: create daemonset/kube-multus-ds-amd64 (apps/v1) namespace: kube-system\n10:31:13AM: ---- waiting on 1 changes [5/6 done] ----\n10:31:13AM: ongoing: reconcile daemonset/kube-multus-ds-amd64 (apps/v1) namespace: kube-system\n10:31:13AM:  ^ Waiting for generation 1 to be observed\n10:31:13AM:  L ongoing: waiting on pod/kube-multus-ds-amd64-r5hcf (v1) namespace: kube-system\n10:31:13AM:     ^ Pending\n10:31:13AM:  L ok: waiting on controllerrevision/kube-multus-ds-amd64-cbf545c97 (apps/v1) namespace: kube-system\n10:31:14AM: ongoing: reconcile daemonset/kube-multus-ds-amd64 (apps/v1) namespace: kube-system\n10:31:14AM:  ^ Waiting for 1 unavailable pods\n10:31:14AM:  L ongoing: waiting on pod/kube-multus-ds-amd64-r5hcf (v1) namespace: kube-system\n10:31:14AM:     ^ Pending: ContainerCreating\n10:31:14AM:  L ok: waiting on controllerrevision/kube-multus-ds-amd64-cbf545c97 (apps/v1) namespace: kube-system\n10:31:16AM: ok: reconcile daemonset/kube-multus-ds-amd64 (apps/v1) namespace: kube-system\n10:31:16AM: ---- applying complete [6/6 done] ----\n10:31:16AM: ---- waiting complete [6/6 done] ----\nSucceeded","updatedAt":"2022-04-25T10:31:16Z"},"fetch":{"exitCode":0,"startedAt":"2022-04-25T10:31:01Z","stdout":"apiVersion: vendir.k14s.io/v1alpha1\ndirectories:\n- contents:\n  - imgpkgBundle:\n      image: projects.registry.vmware.com/tce/multus-cni@sha256:222728c4313e7f5015f5f59fdb8be114e1929b5b1349eaec59d6e710e1bf1395\n    path: .\n  path: \"0\"\nkind: LockConfig\n","updatedAt":"2022-04-25T10:31:11Z"},"friendlyDescription":"Reconcile succeeded","inspect":{"exitCode":0,"stdout":"Target cluster 'https://10.96.0.1:443' (nodes: ucclust-control-plane)\nResources in app 'multus-cni-pkg-ctrl'\nNamespace    Name                                            Kind                      Owner    Conds.  Rs  Ri  Age\n(cluster)    multus                                          ClusterRole               kapp     -       ok  -   5s\n^            multus                                          ClusterRoleBinding        kapp     -       ok  -   5s\n^            network-attachment-definitions.k8s.cni.cncf.io  CustomResourceDefinition  kapp     2/2 t   ok  -   5s\nkube-system  kube-multus-ds-amd64                            DaemonSet                 kapp     -       ok  -   4s\n^            kube-multus-ds-amd64-cbf545c97                  ControllerRevision        cluster  -       ok  -   4s\n^            kube-multus-ds-amd64-r5hcf                      Pod                       cluster  4/4 t   ok  -   4s\n^            multus                                          ServiceAccount            kapp     -       ok  -   5s\n^            multus-cni-config                               ConfigMap                 kapp     -       ok  -   5s\nRs: Reconcile state\nRi: Reconcile information\n8 resources\nSucceeded","updatedAt":"2022-04-25T10:31:17Z"},"observedGeneration":1,"template":{"exitCode":0,"stderr":"resolve | final: ghcr.io/k8snetworkplumbingwg/multus-cni:v3.7.1 -\u003e ghcr.io/k8snetworkplumbingwg/multus-cni@sha256:e72aa733faf24d1f62b69ded1126b9b8da0144d35c8a410c4fa0a860006f9eed\n","updatedAt":"2022-04-25T10:31:12Z"}}
+ kubectl get daemonset kube-multus-ds -n kube-system -o jsonpath={.status}
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

STEP: Should delete Multus CNI test temporary resources
+ kubectl delete -f /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/simple-macvlan.yaml
Error from server (NotFound): error when deleting "/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/simple-macvlan.yaml": network-attachment-definitions.k8s.cni.cncf.io "macvlan-config" not found
Error from server (NotFound): error when deleting "/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/simple-macvlan.yaml": pods "macvlan-worker" not found


• Failure in Spec Setup (BeforeEach) [139.453 seconds]
Multus CNI Addon E2E Test
/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go:26
  Check macvlan interfaces status [BeforeEach]
  /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go:98

  Timed out after 120.004s.
  kube-system/kube-multus-ds daemonset was never ready
  Error: Unexpected non-nil/non-zero extra argument at index 1:
        <*errors.errorString>: &errors.errorString{s:"Error from server (NotFound): daemonsets.apps \"kube-multus-ds\" not found\n\nexit status: 1"}

  /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go:42
------------------------------
Multus CNI Addon E2E Test 
  Check macvlan interfaces status
  /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go:98
STEP: Install Multus CNI Package and tests' resources
+ tanzu package install multus-cni-pkg --package-name multus-cni.community.tanzu.vmware.com --version 3.7.1 -n default -f /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/multus-cni-values.yaml

+ kubectl apply -f /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/cni-install.yaml
configmap/cni-install-sh unchanged
daemonset.apps/install-cni-plugins unchanged

+ kubectl -n default get packageinstalls multus-cni-pkg -o jsonpath={.status.conditions[?(@.type == 'ReconcileSucceeded')].status}
True
+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

+ kubectl -n kube-system get daemonset kube-multus-ds -o jsonpath='{.status.desiredNumberScheduled}'
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found


After test fails, theses resources are related.
+ kubectl get app multus-cni-pkg -n default -o jsonpath={.status}
{"conditions":[{"status":"True","type":"ReconcileSucceeded"}],"consecutiveReconcileSuccesses":2,"deploy":{"exitCode":0,"finished":true,"startedAt":"2022-04-25T10:33:30Z","stdout":"Target cluster 'https://10.96.0.1:443' (nodes: ucclust-control-plane)\nChanges\nNamespace  Name  Kind  Conds.  Age  Op  Op st.  Wait to  Rs  Ri\nOp:      0 create, 0 delete, 0 update, 0 noop\nWait to: 0 reconcile, 0 delete, 0 noop\nSucceeded","updatedAt":"2022-04-25T10:33:31Z"},"fetch":{"exitCode":0,"startedAt":"2022-04-25T10:33:21Z","stdout":"apiVersion: vendir.k14s.io/v1alpha1\ndirectories:\n- contents:\n  - imgpkgBundle:\n      image: projects.registry.vmware.com/tce/multus-cni@sha256:222728c4313e7f5015f5f59fdb8be114e1929b5b1349eaec59d6e710e1bf1395\n    path: .\n  path: \"0\"\nkind: LockConfig\n","updatedAt":"2022-04-25T10:33:30Z"},"friendlyDescription":"Reconcile succeeded","inspect":{"exitCode":0,"stdout":"Target cluster 'https://10.96.0.1:443' (nodes: ucclust-control-plane)\nResources in app 'multus-cni-pkg-ctrl'\nNamespace    Name                                            Kind                      Owner    Conds.  Rs  Ri  Age\n(cluster)    multus                                          ClusterRole               kapp     -       ok  -   2m\n^            multus                                          ClusterRoleBinding        kapp     -       ok  -   2m\n^            network-attachment-definitions.k8s.cni.cncf.io  CustomResourceDefinition  kapp     2/2 t   ok  -   2m\nkube-system  kube-multus-ds-amd64                            DaemonSet                 kapp     -       ok  -   2m\n^            kube-multus-ds-amd64-cbf545c97                  ControllerRevision        cluster  -       ok  -   2m\n^            kube-multus-ds-amd64-r5hcf                      Pod                       cluster  4/4 t   ok  -   2m\n^            multus                                          ServiceAccount            kapp     -       ok  -   2m\n^            multus-cni-config                               ConfigMap                 kapp     -       ok  -   2m\nRs: Reconcile state\nRi: Reconcile information\n8 resources\nSucceeded","updatedAt":"2022-04-25T10:33:31Z"},"observedGeneration":1,"template":{"exitCode":0,"stderr":"resolve | final: ghcr.io/k8snetworkplumbingwg/multus-cni:v3.7.1 -\u003e ghcr.io/k8snetworkplumbingwg/multus-cni@sha256:e72aa733faf24d1f62b69ded1126b9b8da0144d35c8a410c4fa0a860006f9eed\n","updatedAt":"2022-04-25T10:33:30Z"}}
+ kubectl get daemonset kube-multus-ds -n kube-system -o jsonpath={.status}
Error from server (NotFound): daemonsets.apps "kube-multus-ds" not found

STEP: Should delete Multus CNI test temporary resources
+ kubectl delete -f /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/simple-macvlan.yaml
Error from server (NotFound): error when deleting "/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/simple-macvlan.yaml": network-attachment-definitions.k8s.cni.cncf.io "macvlan-config" not found
Error from server (NotFound): error when deleting "/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/simple-macvlan.yaml": pods "macvlan-worker" not found


• Failure in Spec Setup (BeforeEach) [123.178 seconds]
Multus CNI Addon E2E Test
/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go:26
  Check macvlan interfaces status [BeforeEach]
  /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go:98

  Timed out after 120.003s.
  kube-system/kube-multus-ds daemonset was never ready
  Error: Unexpected non-nil/non-zero extra argument at index 1:
        <*errors.errorString>: &errors.errorString{s:"Error from server (NotFound): daemonsets.apps \"kube-multus-ds\" not found\n\nexit status: 1"}

  /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go:42
------------------------------


Summarizing 2 Failures:

[Fail] Multus CNI Addon E2E Test [BeforeEach] Check macvlan interfaces status 
/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go:42

[Fail] Multus CNI Addon E2E Test [BeforeEach] Check macvlan interfaces status 
/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go:42

Ran 1 of 1 Specs in 265.419 seconds
FAIL! -- 0 Passed | 1 Failed | 0 Flaked | 0 Pending | 0 Skipped
--- FAIL: TestMultusCNIE2E (265.42s)
FAIL

Ginkgo ran 1 suite in 4m27.552483821s
Test Suite Failed
make: *** [e2e-test] Error 1
```

--- a/addons/packages/multus-cni/3.7.1/test/e2e/multihomed-testfiles/cni-install.yaml
+++ b/addons/packages/multus-cni/3.7.1/test/e2e/multihomed-testfiles/cni-install.yaml
@@ -41,10 +41,10 @@ spec:
         resources:
           requests:
             cpu: "20m"
-            memory: "50Mi"
+            memory: "100Mi"
           limits:
             cpu: "100m"
-            memory: "50Mi"
+            memory: "100Mi"
         securityContext:
           privileged: true
         volumeMounts:
@@ -52,6 +52,17 @@ spec:
           mountPath: /host/opt/cni/bin
         - name: scripts
           mountPath: /scripts
+        startupProbe:
+          initialDelaySeconds: 10
+          periodSeconds: 10
+          timeoutSeconds: 120
+          successThreshold: 1
+          failureThreshold: 10
+          exec:
+            command:
+            - test
+            - -e
+            - /host/opt/cni/bin/macvlan
       volumes:
         - name: cni-bin
           hostPath:

--- a/addons/packages/multus-cni/3.7.1/test/e2e/multuscni_suite_test.go
+++ b/addons/packages/multus-cni/3.7.1/test/e2e/multuscni_suite_test.go
@@ -42,7 +42,7 @@ func TestMultusCNIE2E(t *testing.T) {
 var _ = BeforeSuite(func() {
 	// Needs to have package related values predefined.
 	packageName = utils.TanzuPackageName("multus-cni")
-	packageFullVersion = utils.TanzuPackageAvailableVersion(packageName)
+	packageFullVersion = utils.TanzuPackageAvailableVersionWithVersionSubString(packageName, packageVersion)
 	packageInstalledName = "multus-cni-pkg"
 	packageInstalledNamespace = "default"
 	if strings.Compare(packageInstalledNamespace, "default") != 0 {

--- a/addons/packages/multus-cni/3.7.1/test/e2e/multuscni_test.go
+++ b/addons/packages/multus-cni/3.7.1/test/e2e/multuscni_test.go
@@ -35,15 +35,15 @@ var _ = Describe("Multus CNI Addon E2E Test", func() {
 		}
 		_, err = utils.Tanzu(nil, installOptions...)
 		Expect(err).NotTo(HaveOccurred())
-		_, err = utils.Kubectl(nil, "apply", "-f", filepath.Join(curDir, "multihomed-testfiles/cni-install.yaml"))
-		Expect(err).NotTo(HaveOccurred())
-
 		utils.ValidatePackageInstallReady(packageInstalledNamespace, packageInstalledName)
 		utils.ValidateDaemonsetReady("kube-system", "kube-multus-ds-amd64")
+
+		_, err = utils.Kubectl(nil, "apply", "-f", filepath.Join(curDir, "multihomed-testfiles/cni-install.yaml"))
+		Expect(err).NotTo(HaveOccurred())
+		utils.ValidateDaemonsetReady("kube-system", "install-cni-plugins")
+
 		_, err = utils.Kubectl(nil, "apply", "-f", filepath.Join(curDir, "multihomed-testfiles/simple-macvlan.yaml"))
 		Expect(err).NotTo(HaveOccurred())
-
-		utils.ValidateDaemonsetReady("kube-system", "install-cni-plugins")
 		utils.ValidatePodReady("default", "macvlan-worker")
 
 		// Just do a simple check for which image is using
@@ -66,6 +66,8 @@ var _ = Describe("Multus CNI Addon E2E Test", func() {
 			_, err := utils.Kubectl(nil, "get", "app", packageInstalledName, "-n", packageInstalledNamespace, "-o", "jsonpath={.status}")
 			Expect(err).NotTo(HaveOccurred())
 			_, err = utils.Kubectl(nil, "get", "daemonset", "kube-multus-ds-amd64", "-n", "kube-system", "-o", "jsonpath={.status}")
+			Expect(err).NotTo(HaveOccurred())
+			_, err = utils.Kubectl(nil, "get", "pod", "macvlan-worker", "-n", "default", "-o", "jsonpath={.status}")
 			Expect(err).NotTo(HaveOccurred())
 		}
 	})

--- a/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/cni-install.yaml
+++ b/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/cni-install.yaml
@@ -41,10 +41,10 @@ spec:
         resources:
           requests:
             cpu: "20m"
-            memory: "50Mi"
+            memory: "100Mi"
           limits:
             cpu: "100m"
-            memory: "50Mi"
+            memory: "100Mi"
         securityContext:
           privileged: true
         volumeMounts:
@@ -52,6 +52,17 @@ spec:
           mountPath: /host/opt/cni/bin
         - name: scripts
           mountPath: /scripts
+        startupProbe:
+          initialDelaySeconds: 10
+          periodSeconds: 10
+          timeoutSeconds: 120
+          successThreshold: 1
+          failureThreshold: 10
+          exec:
+            command:
+            - test
+            - -e
+            - /host/opt/cni/bin/macvlan
       volumes:
         - name: cni-bin
           hostPath:

--- a/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/multus-cni-values.yaml
+++ b/addons/packages/multus-cni/3.8.0/test/e2e/multihomed-testfiles/multus-cni-values.yaml
@@ -1,0 +1,6 @@
+namespace: kube-system
+
+daemonset:
+  args:
+    - "--multus-conf-file=auto"
+    - "--cni-version=0.3.1"

--- a/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_suite_test.go
+++ b/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_suite_test.go
@@ -42,7 +42,7 @@ func TestMultusCNIE2E(t *testing.T) {
 var _ = BeforeSuite(func() {
 	// Needs to have package related values predefined.
 	packageName = utils.TanzuPackageName("multus-cni")
-	packageFullVersion = utils.TanzuPackageAvailableVersion(packageName)
+	packageFullVersion = utils.TanzuPackageAvailableVersionWithVersionSubString(packageName, packageVersion)
 	packageInstalledName = "multus-cni-pkg"
 	packageInstalledNamespace = "default"
 	if strings.Compare(packageInstalledNamespace, "default") != 0 {

--- a/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go
+++ b/addons/packages/multus-cni/3.8.0/test/e2e/multuscni_test.go
@@ -35,15 +35,15 @@ var _ = Describe("Multus CNI Addon E2E Test", func() {
 		}
 		_, err = utils.Tanzu(nil, installOptions...)
 		Expect(err).NotTo(HaveOccurred())
-		_, err = utils.Kubectl(nil, "apply", "-f", filepath.Join(curDir, "multihomed-testfiles/cni-install.yaml"))
-		Expect(err).NotTo(HaveOccurred())
-
 		utils.ValidatePackageInstallReady(packageInstalledNamespace, packageInstalledName)
 		utils.ValidateDaemonsetReady("kube-system", "kube-multus-ds")
+
+		_, err = utils.Kubectl(nil, "apply", "-f", filepath.Join(curDir, "multihomed-testfiles/cni-install.yaml"))
+		Expect(err).NotTo(HaveOccurred())
+		utils.ValidateDaemonsetReady("kube-system", "install-cni-plugins")
+
 		_, err = utils.Kubectl(nil, "apply", "-f", filepath.Join(curDir, "multihomed-testfiles/simple-macvlan.yaml"))
 		Expect(err).NotTo(HaveOccurred())
-
-		utils.ValidateDaemonsetReady("kube-system", "install-cni-plugins")
 		utils.ValidatePodReady("default", "macvlan-worker")
 
 		// Just do a simple check for which image is using
@@ -66,6 +66,8 @@ var _ = Describe("Multus CNI Addon E2E Test", func() {
 			_, err := utils.Kubectl(nil, "get", "app", packageInstalledName, "-n", packageInstalledNamespace, "-o", "jsonpath={.status}")
 			Expect(err).NotTo(HaveOccurred())
 			_, err = utils.Kubectl(nil, "get", "daemonset", "kube-multus-ds", "-n", "kube-system", "-o", "jsonpath={.status}")
+			Expect(err).NotTo(HaveOccurred())
+			_, err = utils.Kubectl(nil, "get", "pod", "macvlan-worker", "-n", "default", "-o", "jsonpath={.status}")
 			Expect(err).NotTo(HaveOccurred())
 		}
 	})

